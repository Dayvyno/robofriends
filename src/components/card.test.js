import {shallow} from 'enzyme';
import Card from './card'
import React from 'react'

it("Testing the card component", ()=>{
  expect(shallow(<Card />)).toMatchSnapshot()
})