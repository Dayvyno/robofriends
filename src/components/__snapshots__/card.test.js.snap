// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing the card component 1`] = `
<div>
  <div
    className="robohash"
  >
    <p>
      Name: 
    </p>
    <img
      alt="undefined"
      src="https://robohash.org/undefined?size=200x200"
    />
    <p>
      Email:
       
    </p>
  </div>
</div>
`;
